libnavui-maneuver/api/current.txt

```diff
@@ -408,28 +409,35 @@ package com.mapbox.navigation.ui.maneuver.model {
   }
 
   public final class RoadShield {
+    ctor public RoadShield(String shieldUrl, byte[] shieldIcon, com.mapbox.api.directions.v5.models.MapboxShield? mapboxShield = null);
     ctor public RoadShield(String shieldUrl, byte[] shieldIcon);
     method public String component1();
     method public byte[] component2();
-    method public com.mapbox.navigation.ui.maneuver.model.RoadShield copy(String shieldUrl, byte[] shieldIcon);
+    method public com.mapbox.api.directions.v5.models.MapboxShield? component3();
+    method public com.mapbox.navigation.ui.maneuver.model.RoadShield copy(String shieldUrl, byte[] shieldIcon, com.mapbox.api.directions.v5.models.MapboxShield? mapboxShield);
```

Comment:
We still have that `copy` method breaking SemVer. Don't know how serious this is. 
But I don't think there is a way around it, except for creating a new non-data class.

Reply:
`mapboxShield` is the last argument in the last, would that actually break the old copy function?

Reply:
https://jakewharton.com/public-api-challenges-in-kotlin/#copy-functions 😢